legend-depot-pure-model-context/src/main/java/org/finos/legend/depot/server/pure/model/context/resources/PureModelContextResource.java

```diff
@@ -50,17 +52,18 @@ public PureModelContextResource(PureModelContextService service)
     @Path("projects/{groupId}/{artifactId}/versions/{versionId}/pureModelContextData")
     @ApiOperation(GET_VERSION_ENTITIES_AS_PMCD)
     @Produces(MediaType.APPLICATION_JSON)
-    public PureModelContextData getPureModelContextData(@PathParam("groupId") String groupId,
-                                                        @PathParam("artifactId") String artifactId,
-                                                        @PathParam("versionId") @ApiParam(value = VersionValidator.VALID_VERSION_ID_TXT)  String versionId,
-                                                        @QueryParam("clientVersion") String clientVersion,
-                                                        @QueryParam("versioned")
+    public Response getPureModelContextData(@PathParam("groupId") String groupId,
```

Comment:
you should not need all the parameters, etag is specific to a complete URL with params
so a common etag between ?transitive=true and transitive=false is fine and will work as expected.

It may be worth adding a cache busting constant however incase we roll out a backward incompatible change. This could be the platform version (less etag matching) or a constant that is only changed if a backward incompatible change is released (more etag matching)